ember-app/app/models/new/board.js

```diff
@@ -92,6 +92,16 @@ var Board = Model.extend({
       return combined;
     }
   }),
+  issuesById: function(){
+    return _.groupBy(this.get('issues'), (issue)=> {
+      return issue.get('id');
+    });
+  }.property('issues.[]'),
+  issuesByRepo: function(){
+    return _.groupBy(this.get('issues'), (issue)=> {
```

Comment:
So, this is kind of a cheat, but I would like the `issuesByRepo` for other parts of the app as well where I think we could get a performance benefit.